ci-operator/config/openshift/image-customization-controller/openshift-image-customization-controller-main.yaml

```diff
@@ -0,0 +1,27 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+promotion:
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: e2e-aws
```

Comment:
Why `e2e-aws` and not our usual metal-ipi tests?

Reply:
Hi @andfasano 
So I am not very clued up on these jobs.. But I ran "make new-repo" and it added it.
Also I looked at  cbo: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-master.yaml#L128-L131
and it does not have an aws job in CI

Reply:
Not sure why that is there in the CBO, but given that it is a CVO Operator (i.e. can block the rollout completing) maybe it's there to make sure we don't screw up other platforms... which is a thing that has happened iirc.
I don't think the same applies here, so there's reason to run an AWS job for this repo imho. I get why it's in the default set, but this repo is very platform-specific.

Reply:
I'll add unit and lint tests for now (no point in e2e until we have images and we are running the controller

Reply:
we will need to merge this first to get the makefile https://github.com/openshift/image-customization-controller/pull/2

Reply:
"make new repo" => fine didn't realize it :) we could add all the missing baremetal required tests later. I think it's the reasonable default for creating a new OpenShift component repo and yes, we kept it in CBO to be sure to not screw up other platforms, but if it's not strictly required here then we could remove it to save some resources.

/lgtm

Reply:
/hold still some config to be fixed